code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -3,15 +3,59 @@
 /obj/item/weapon/reagent_containers/glass/bottle
 	name = "bottle"
 	desc = "A small bottle."
-	icon_state = null
+	icon_state = "bottle"
 	item_state = "atoxinbottle"
 	possible_transfer_amounts = list(5,10,15,25,30)
 	volume = 30
 
 /obj/item/weapon/reagent_containers/glass/bottle/New()
 	..()
 	if(!icon_state)
-		icon_state = "bottle[rand(1,20)]"
+		icon_state = "bottle"
+	update_icon()
+
+
+/obj/item/weapon/reagent_containers/glass/bottle/on_reagent_change()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/glass/bottle/pickup(mob/user)
+	..()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/glass/bottle/dropped(mob/user)
+	..()
+	update_icon()
```

Comment:
I just took it from the beakers, will test it.

Does throwing items count as dropped?